app/models/middleware_server.rb

```diff
@@ -1,5 +1,6 @@
 class MiddlewareServer < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :lives_on, :polymorphic => true
```

Comment:
I see that you mimic the implementation as on `ContainerNode`, but right now it's only a `Vm`.
Will that change? Otherwise this could be `belongs_to :vm`